profile/README.md

```diff
@@ -0,0 +1,71 @@
+<picture style="display: block; max-width: 100%">
+  <source media="(prefers-color-scheme: dark)" srcset="../artwork/transparency-dev-dark.png">
+  <source media="(prefers-color-scheme: light)" srcset="../artwork/transparency-dev-light.png">
+  <img alt="Transparency.dev logo" src="../artwork/transparency-dev-light.png">
+</picture>
+<br/>
+
+Welcome to transparency.dev, a community dedicated to promoting transparency technologies across various ecosystems and pioneering its integration into new application domains.
+
+Tamper-evident logs are the heart of transparency technology and are relied on by [Certificate Transparency](https://certificate.transparency.dev/), [Sigstore (Supply Chain Security)](https://www.sigstore.dev/), and many other ecosystems. Visit [transparency.dev](https://transparency.dev/) to learn more!
+
+## 🔨 Transparency.dev Projects
+
+This repository hosts critical projects for building end-to-end transparency ecosystems, including:
+- [trillian-tessera](https://github.com/transparency-dev/trillian-tessera) - a Go library for building tile-based transparency logs
+- [static-ct](https://github.com/transparency-dev/static-ct) - An implementation of the Static CT API based on Tessera
+- [witness](https://github.com/transparency-dev/witness) - Libraries and binaries for running witnesses for verifiable logs
+- [armored-witness](https://github.com/transparency-dev/armored-witness) - A plug and play witness solution intended to kick-start a cross-ecosystem witness network
```

Comment:
I think it's worth adding `merkle` and possibly `formats` here too as they're pretty central to everything transparency.dev is about.